client/ayon_core/modules/deadline/plugins/publish/submit_nuke_deadline.py

```diff
@@ -115,8 +115,11 @@ def process(self, instance):
         render_path = instance.data['path']
         script_path = context.data["currentFile"]
 
-        use_published_workfile = instance.data["attributeValues"].get(
-            "use_published_workfile", self.use_published_workfile
+        use_published_workfile = instance.data.get(
```

Comment:
Why this changed. Just set `"use_published_workfile"` on `"attributeValues"` where you change it...